Jenkinsfile

```diff
@@ -7,13 +7,14 @@ pipeline {
     }
     stages {
         stage('Build') {
+            agent { docker { image "python:3" } }
             steps {
                 echo 'Building..'
                 sh '''
-                    virtualenv ./sdk-deploy
+                    rm -r dist
```

Comment:
Failed jobs don't remove the dist dir. This will force the directory to empty even if the previous job fails.